src/modules/m_disable.cpp

```diff
@@ -116,7 +116,7 @@ class ModuleDisable : public Module
 		commands.swap(newcommands);
 
 		// Whether we should fake the non-existence of disabled things.
-		fakenonexistent = tag->getBool("fakenonexistent", tag->getBool("fakenonexistant"));
+		fakenonexistent = tag->getBool("fakenonexistent", tag->getBool("fakenonexistent"));
```

Comment:
ok, this is wrong. What is this code trying to do?

Reply:
This is a previous typo fix for a config variable, so we still accept the former (incorrect) spelling to not break configs.